src/components/Multiselect/Multiselect.vue

```diff
@@ -41,7 +41,7 @@
       v-show="open"
       role="listbox"
       :aria-expanded="open"
-      class="absolute rounded border border-gray-100 mt-0.5 w-full left-0"
+      class="absolute rounded border border-gray-100 mt-0.5 w-full left-0 bg-white z-30"
```

Comment:
Would this still work if we just did `z-30`? Would love to keep a color out of it if possible so it can be more flexible in the future if/when we open source so folks can use their own color schemes.

Reply:
I _believe_ it is needed, as the background color would be transparent if not! Does that make sense or am I missing something?  I just checked `dropdown` and it uses a `bg-white` too
![image](https://user-images.githubusercontent.com/83967528/134030945-c5c371ea-29ec-4f4a-9d66-5ac9d2c4633b.png)

Reply:
That makes sense! Sounds good. ☺️